### PR TITLE
Use node-uuid over Date.now()

### LINKED ISSUE #71: Use UUID for id of todoItems

### PR DESCRIPTION
**Pull request by agubler**
Use unique identifiers for the todo ids

Fixes #71

### CODE REVIEW

File: todo-mvc/typings.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
 	"dependencies": {
 		"@reactivex/RxJS": "npm:@reactivex/rxjs",
 		"immutable": "npm:immutable",
-		"maquette": "npm:maquette"
+		"maquette": "npm:maquette",
+		"node-uuid": "registry:npm/node-uuid#1.4.7+20160723033700"
```
Comment (kitsonk) [inline todo-mvc/typings.json]: There is [@types/node-uuid](https://www.npmjs.com/package/@types/node-uuid).

Comment (agubler) [inline todo-mvc/typings.json]: I did initially use `@types/node-uuid` and it made a complete mess and caused lots of problems in clashes with existing typings.

**Comment by matt-gadd** [comment]
@kitsonk @agubler I thought we'd discussed this before :) https://github.com/dojo/widgets/issues/20

**Comment by agubler** [comment]
I'm not sure it's the same, we were using Date.now() to create ids for the todo items so this replaces it with a UUID.

**Comment by matt-gadd** [comment]
@agubler i meant the discussion on whether a guid generator should live in core.

**Comment by rishson** [comment]
GUID gen in core is a useful addition IMO, as long as our impl is robust (meets 4122 compliance)

**Comment by agubler** [comment]
Ah okay, carry on :)

**Comment by kitsonk** [comment]
dojo/core#214 is now open to implement it, obviously if it is expedient to still land this for now, I am fine, but I still think it is something we should focus on having as part of core.

**Comment by agubler** [comment]
Closed in favour of waiting for a `dojo-core` alternative

**Pull request closed without merging**